aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/LambdaHTTPResponse.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.amazonaws.services.lambda.runtime.events;
+
+import lombok.ToString;
+
+@ToString(callSuper = true)
+public class LambdaHTTPResponse extends APIGatewayV2HTTPResponse {
```

Comment:
Suggest renaming to `LambdaFunctionURLResponse` to align with the feature name (ref. https://aws.amazon.com/about-aws/whats-new/2022/04/aws-lambda-function-urls-built-in-https-endpoints/) - the Go lib will be using this name